compass/ocean/tests/parabolic_bowl/viz/__init__.py

```diff
@@ -231,19 +232,20 @@ def rmse_plots(self):
                 if rmse[i] > max_rmse:
                     max_rmse = rmse[i]
 
-            ax.loglog(self.resolutions, rmse,
+            ax.loglog(resolutions, rmse,
                       linestyle='-', marker='o', label=comp)
 
-        rmse_1st_order = np.zeros(len(self.resolutions))
+        rmse_1st_order = np.zeros(len(resolutions))
         rmse_1st_order[0] = max_rmse
-        for i in range(len(self.resolutions) - 1):
+        for i in range(len(resolutions) - 1):
             rmse_1st_order[i + 1] = rmse_1st_order[i] / 2.0
 
-        ax.loglog(self.resolutions, rmse_1st_order,
+        ax.loglog(resolutions, rmse_1st_order,
                   linestyle='-', color='k', alpha=.25, label='1st order')
 
         ax.set_xlabel('Cell size (km)')
         ax.set_ylabel('RMSE (m)')
+        ax.invert_xaxis()
```

Comment:
@cbegeman, this is the only real substantive change here, correct? The `self.resolutions` replacements are a nice cleanup, though. Thanks for fixing this!

Reply:
Yep. I originally thought we also needed to sort `resolutions` but that ended up not being necessary.